Formatting and spec clarity updates (#7)
OR13/draft-steele-cose-hash-envelope: draft-steele-cose-hash-envelope.md

--- abstract

> This document defines new COSE header parameters supporting of hashes within the payload, representing a detached COSE payloads.

Reviewer: ```suggestion
This document defines new COSE header parameters for signaling that a payload is the output of a hash function.
```

Author: It likely needs a bit more tweaks for readability, but it's more accurate so merged into the PR for content tweaking later.
